antlr_config/ManimParser.g4

```diff
@@ -4,35 +4,52 @@ options {
   tokenVocab=ManimLexer;
 }
 
-program: function* (stat SEMI)+ EOF;
+program: function* stat EOF;
 
-function: FUN IDENT OPEN_PARENTHESIS param_list? CLOSE_PARENTHESIS (COLON type)? OPEN_CURLY_BRACKET (stat SEMI)* CLOSE_CURLY_BRACKET;
+function: FUN IDENT OPEN_PARENTHESIS param_list? CLOSE_PARENTHESIS (COLON type)? OPEN_CURLY_BRACKET statements=stat? CLOSE_CURLY_BRACKET;
 
 param_list: param (COMMA param)*                                    #ParameterList;
 
 param: type IDENT                                                   #Parameter;
 
-stat: SLEEP OPEN_PARENTHESIS expr CLOSE_PARENTHESIS                 #SleepStatement
-    | COMMENT OPEN_PARENTHESIS STRING CLOSE_PARENTHESIS             #CommentStatement // when string type defined we can adjust
-    | LET IDENT (COLON type)? EQUAL expr                            #DeclarationStatement
-    | IDENT EQUAL expr                                              #AssignmentStatement
-    | method_call                                                   #MethodCallStatement
-    | RETURN expr                                                   #ReturnStatement;
+stat: SLEEP OPEN_PARENTHESIS expr CLOSE_PARENTHESIS ';'                 #SleepStatement
+    | COMMENT OPEN_PARENTHESIS STRING CLOSE_PARENTHESIS ';'             #CommentStatement // when string type defined we can adjust
+    | LET IDENT (COLON type)? EQUAL expr ';'                            #DeclarationStatement
+    | IDENT EQUAL expr ';'                                              #AssignmentStatement
```

Comment:
Can we use `SEMI` instead?

Reply:
I would prefer to keep them as literal tokens because it does help with readability of the grammar